Packages/com.chisel.editor/Chisel/Editor/ComponentEditors/Base/ChiselEditableOutline.cs

```diff
@@ -901,7 +901,7 @@ ClickState HandleClick(int id, bool captureControl = false)
                     if (captureControl)
                     {
                         SetControl(evt, 0);
-                        Snapping.ActiveAxes = Axes.XYZ;
+                        UnitySceneExtensions.Snapping.ActiveAxes = Axes.XYZ;
```

Comment:
fyi; 
this could've also been solved with
using Snapping = UnitySceneExtensions.Snapping;

Reply:
I'm fine with solving it this way, but just mentioning it that in the future it might be better to solve naming collisions using "using", or renaming the class .. otherwise things get so verbose